changelog/unreleased/4423

```diff
@@ -0,0 +1,6 @@
+Enhancement: Changes in the Feedback section
+
+Based on a brandable parameter, a new dialog has been added to handle feedback. Within the dialog, links to GitHub and the open forum Central will be displayed.
```

Comment:
Now we must mention the link to the survey as well, which is the main element there

Reply:
Updated